src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -1593,181 +1593,434 @@ void UpdateZoneSizing(EnergyPlusData &state, DataGlobalConstants::CallIndicator
     Real64 SupplyTemp;     // supply air temperature [C]
     Real64 DeltaTemp;      // supply air delta temperature [deltaC]
 
-    {
-        auto const SELECT_CASE_var(CallIndicator);
-
-        if (SELECT_CASE_var == DataGlobalConstants::CallIndicator::BeginDay) {
-
-            for (CtrlZoneNum = 1; CtrlZoneNum <= state.dataGlobal->NumOfZones; ++CtrlZoneNum) {
+    switch (CallIndicator) {
+    case DataGlobalConstants::CallIndicator::BeginDay: {
+        for (CtrlZoneNum = 1; CtrlZoneNum <= state.dataGlobal->NumOfZones; ++CtrlZoneNum) {
```

Comment:
Reduce scope of loop induction variables to loop itself: `for (int CtrlZoneNum ...`.